g2p_registry_membership/models/individual.py

```diff
@@ -17,8 +17,12 @@ def _recompute_parent_groups(self, records):
         field = self.env["res.partner"]._fields["force_recompute_canary"]
         for line in records:
             if line.is_registrant and not line.is_group:
-                groups = line.individual_membership_ids.mapped("group")
-                self.env.add_to_compute(field, groups)
+                try:
+                    groups = line.individual_membership_ids.mapped("group")
+                except Exception as e:
+                    _logger.info("_recompute_parent_groups: exception: %s" % e)
```

Comment:
Can this be a warning at least? Because if it happen it will not recompute the parent which is an issue.